src/images.ts

```diff
@@ -8,3 +8,10 @@ import "./assets/form-field-error-icon.svg";
 import "./assets/no-users-graphic.svg";
 import "./assets/chevron-left.svg";
 import "./assets/chevron-right.svg";
+import "./assets/right_arrow_icon.svg"
+import "./assets/star_sparkle_stars_sparkles_icon.svg"
+import "./assets/supertokens.svg"
+import "./assets/supertokens-icon.svg"
+import "./assets/supertokens-dark.svg"
```

Comment:
Use images from https://github.com/supertokens/supertokens-logo/tree/master/logo for this.

Dont add the asset files, load the images using the urls

Reply:
its updated